test/Microsoft.Azure.EventHubs.UnitTests/EventHubClientTests.cs

```diff
@@ -655,6 +655,62 @@ void ValidateRetryPolicyCustom()
                 $"Didn't receive the same number of messages that we sent. Sent: {NumberOfMessagesToSend}, Received: {totalReceived}");
         }
 
+        [Fact]
```

Comment:
I am not sure why these tests show as diff since these are already merged as part of previous PR.